### ISSUE TITLE
Add PrimAD reference data

### ISSUE THREAD

**Issue by hanars** [broadinstitute/seqr-loading-pipelines#320]
From the original ticket in the seqr repo https://github.com/broadinstitute/seqr/issues/2547 :

primAD has sequence data from ~1000 primates from many different species and is helpful to look at variation in this database, similar to looking at gnomAD. We would want to know the primate AF for a variant, as this is some evidence that the variant is tolerated. In these 1000 primates, there is as much variation seen as there is in the 60,000 people in ExAC.

Add AC/AFs from primAD as a reference population dataset, with link to the primAD variant page - BUT only for the primate data. primAD also has a copy of the gnomAD data which we already have in seqr, so we would want the data just for primates.

**Comment by mattsolo1** [comment]
I'm not aware of a bulk data download, but you can fetch variant data using the API, e.g.

https://primad.basespace.illumina.com/primateApi/variant?bvid=1-55046543-C-T

**Comment by hanars** [comment]
So the issue is we only want to show a link if the variant actually exists in primad, if we just added that style link to every variant in seqr many of them would go nowhere. So we do need a bulk download in order to know if the variant is in primad or not (and then AF/AC would be nice to have but not essential)

**Comment by hanars** [comment]
Yeah I would rather not do that in seqr, I'm worried on a page with 100 variants theres going to be a lag and we'll end up with links popping up on a variant after an analyst already scrolled past it. I think we should hold off until we can get some sort of download, or maybe do those requests as part of the loading?

**Comment by mike-w-wilson** [comment]
Need to contact primAD to see if they have a download available

**Comment by lynnpais** [comment]
Talked to Anne and decided to hold off on this as its not being used much.

**Comment by hanars** [comment]
Blocked on needing a bulk download
